runtime/src/main/scala/io/github/touchdown/gyremock/GyremockSettings.scala

```diff
@@ -15,4 +20,4 @@ object GyremockSettings {
     )
 }
 
-case class GyremockSettings(host: String, port: Int, stopTimeout: FiniteDuration, wiremockBaseUrl: Option[String])
+case class GyremockSettings(host: String = localhost, port: Int = dynamicPort, stopTimeout: FiniteDuration = 10.seconds, wiremockBaseUrl: Option[String])
```

Comment:
i meant, we should not have defaults here and just pass in the values explicitly in test

Reply:
Yeah. However, I think a server could provide convenience if we feel some use cases are common. And defaults could be convenient. In the current situation, I think the most common use case is starting a local-host server with a dynamic port, so I provide some defaults here. But I think it is acceptable to avoid providing defaults in most situations.

By the way, I have a generated question that I want to learn: How do you think about defaults? Which situations are suitable and which are not?

Reply:
Make sense. Will remove these default values